### ISSUE TITLE
get session or record id

### ISSUE THREAD

**Issue by vinibgoulart** [microsoft/clarity#508]
is there a way to get the session or record id? i want to sync this id with another software i have, but i dont want to generate the id by my side, i would like to get clarity session or record id

**Comment by ranjanamaurya06** [comment]
Yes, there is an API you can use to get MS Clarity session information, such as user ID, etc. From my experience, this wasn't documented clearly, but after exploring the GitHub repository, I found this method, which works well for my use case. I hope this helps!

```
window.clarity("metadata", ({ projectId, userId, sessionId }) => {
  // logic goes here
}, false);
```

**Comment by ClaritySupport** [comment]
Thanks @ranjanamaurya06 for sharing the snippet
